types/src/transaction/mod.rs

```diff
@@ -205,6 +244,53 @@ impl RawTransaction {
         }
     }
 
+    pub fn new_txn(
+        sender: AccountAddress,
+        replay_protector: ReplayProtector,
+        executable: TransactionExecutable,
+        multisig_address: Option<AccountAddress>,
+        max_gas_amount: u64,
+        gas_unit_price: u64,
+        expiration_timestamp_secs: u64,
+        chain_id: ChainId,
+    ) -> Self {
+        match replay_protector {
+            ReplayProtector::SequenceNumber(sequence_number) => RawTransaction {
+                sender,
+                sequence_number,
+                payload: TransactionPayload::V2(TransactionPayloadV2::V1 {
```

Comment:
V2 probably here needs to be gated until V2 format is enabled?

or you have a separate PR to support V2 format and it's processing, and then a later PR (in a later release) to use it throughout